style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -23,6 +23,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     themeable: true,
     public: true,
   },
+  colorBackgroundToggleButtonNormalPressed: {
```

Comment:
Why is only the pressed state of the toggle button themeable?

Reply:
because they inherit all tokens from the button component